lib/cli.js

```diff
@@ -90,17 +94,17 @@ var processArgs = function (argv, options, fs, path) {
 
   if (!configFile) {
     // default config file (if exists)
-    if (fs.existsSync('./karma.conf.js')) {
+    if (helper.existsSync('./karma.conf.js')) {
```

Comment:
`fs.exists()` has been deprecated) a while.  Fixed this while I was in there.  I added it to the helpers, wrapping `fs.accessSync` and checking if the process has read access to the file.

https://nodejs.org/api/fs.html#fs_fs_existssync_path